tests/KdybyTests/Doctrine/Console/GenerateEntitiesCommand.phpt

```diff
@@ -34,12 +34,15 @@ class GenerateEntitiesCommandTest extends CommandTestCase
 		]);
 
 		$output = $applicationTester->getDisplay();
+		$output = str_replace('  ', '', $output);
+		$output = preg_replace( "/\r|\n/", '', $output );
 
 		foreach (self::$entities as $entity) {
 			Assert::contains("Processing entity \"{$entity}\"", $output);
 		}
 		Assert::notContains(sprintf('Processing entity "%s"', \KdybyTests\Doctrine\Models2\Foo::class), $output);
-		Assert::contains('Entity classes generated to "' . realpath($destDir) . '"', $output);
+		Assert::contains('Entity classes generated to', $output);
+		Assert::contains(realpath($destDir), $output);
```

Comment:
What is different about the output now that makes the old assert invalid?

Reply:
it was before fix output, I rollbacked it ;)